classes/mcPy/McPy.py

```diff
@@ -49,6 +49,8 @@ def _launch(parser: Parser):
 
     # Here starts the server :D
     server = Server.Server(parser, avail_cores)
+    if parser.debug:
+        logging.basicConfig(level=logging.DEBUG, format='[%(asctime)s - %(levelname)s - %(threadName)s] %(message)s', force=True)
```

Comment:
See #72 & #73. I prefer reverting #73 over this

Reply:
i will revert it if #73/#72 gets reverted